shm/Fortune.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+#include <exception>
+#include <random>
+#include <stdexcept>
+#include <vector>
+
+struct Fortune {
```

Comment:
This whole class is static. So probably we should create a standalone functions instead?